reviews/graduation-jaeger.md

```diff
@@ -0,0 +1,156 @@
+# Jaeger Graduation Application
+
+Jaeger was open sourced in April, 2017 and joined the CNCF in September, 2017. The project has grown significantly over time. As of Apr-22-2019 ([devstats][devstats]):
+
+  * 7806 stars on the main repository https://github.com/jaegertracing/jaeger
+  * 1260 forks
+  * 321 authors of commits and pull requests
+  * 515 issue creators
+
+Jaeger fulfills all CNCF incubating and graduation criteria. The following application links to the required information to become a graduated project.
+
+## Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+Known end users include: Alibaba Cloud, Base CRM (acquired by ZenDesk), Candide, Circonus, ContaAzul, FarmersEdge, GrafanaLabs, Massachusetts Open Cloud, Northwestern Mutual, Nets, Stagemonitor, RiksTV, SeatGeek, SpotHero, Ticketmaster, Vistar Media, Uber, Weave HQ, Weaveworks, Zenly.
+
+Some specific examples:
+
+  * [Uber Technologies](https://eng.uber.com/distributed-tracing/)
+    * Running Jaeger in production since 2015.
+    * Over 2700 microservices are integrated with Jaeger, which collects about 10 billion spans per day, across multiple data centers.
+    * Data mining techniques on traces are used to significantly reduce root cause analysis time during outages.
+    * Tracing data is utilized by a number of other internal tools, such as using it to reduce duplicate alerts, and to infer data lineage.
+  * [Weaveworks](https://www.weave.works/) empowers developers and DevOps teams to build better software faster. Using Jaeger in production since Spring 2018. ~70 microservices, ~600 containers, most of services are instrumented with Jaeger. Using AWS Elasticsearch as the span storage backend, with two weeks retention. 
```

Comment:
@monadic do you have anything that you want to add here in support of graduation from the Weaveworks perspective?